package.json

```diff
@@ -17,7 +17,8 @@
     "stop:all": "npm stop; npm run stop:mockapi",
     "chimp": "NODE_ENV=acceptance chimp --mocha --browser=htmlunit --path=./test/acceptance/spec/",
     "chimp:dev": "NODE_ENV=acceptance chimp --mocha --path=./test/acceptance/ --watch --browser=firefox",
-    "lint": "eslint .",
+    "lint": "node_modules/.bin/eslint $FIXLINT .",
```

Comment:
fyi you don't need the .bin in there npm run script will do that for you:

https://docs.npmjs.com/cli/run-script

```In addition to the shell's pre-existing PATH, npm run adds node_modules/.bin to the PATH provided to scripts. Any binaries provided by locally-installed dependencies can be used without the node_modules/.bin prefix.```